### PR TITLE
Pull request 235

### PR DESCRIPTION
**Pull request by queglay**
Added optional --skip-package-update arg to bypass yum/apt updates

### CODE REVIEW

File: modules/install-vault/install-vault [hunk 1]
```diff
@@ -10,6 +10,7 @@ set -e
 
 readonly DEFAULT_INSTALL_PATH="/opt/vault"
 readonly DEFAULT_VAULT_USER="vault"
+readonly DEFAULT_SKIP_PACKAGE_UPDATE=false
```
Comment (brikis98) [inline modules/install-vault/install-vault:13]:
```suggestion
readonly DEFAULT_SKIP_PACKAGE_UPDATE="false"
```

File: modules/install-vault/install-vault [hunk 2]
```diff
@@ -30,6 +31,7 @@ function print_usage {
   echo -e "  --download-url\t\tUrl to exact Vault package to be installed. Optional if version is provided."
   echo -e "  --path\t\tThe path where Vault should be installed. Optional. Default: $DEFAULT_INSTALL_PATH."
   echo -e "  --user\t\tThe user who will own the Vault install directories. Optional. Default: $DEFAULT_VAULT_USER."
+  echo -e "  --skip-package-update\t\tSkip yum/apt updates. Optional. Default: $DEFAULT_SKIP_PACKAGE_UPDATE."
```
Comment (brikis98) [inline modules/install-vault/install-vault:34]:
```suggestion
  echo -e "  --skip-package-update\t\tSkip yum/apt updates. Optional. Only recommended if you already ran yum update or apt-get update yourself. Default: $DEFAULT_SKIP_PACKAGE_UPDATE."
```

File: modules/install-vault/install-vault [hunk 3]
```diff
@@ -239,6 +248,10 @@ function install {
         user="$2"
         shift
         ;;
+      --skip-package-update)
+        skip_package_update="$2"
+        shift
```
Comment (brikis98) [inline modules/install-vault/install-vault:253]:
```suggestion
        skip_package_update="true"
```

**Review by brikis98 — COMMENTED** [review]
Thanks for the PR!

**Review by brikis98 — APPROVED** [review]
LGTM, thanks! I'll kick off tests now.

**Comment by brikis98** [comment]
Ah, tests failed, but due to an unrelated reason: `ap-northeast-3` doesn't have the dedicated KMS key we set up for automated tests in our AWS account. I know it's not part of this PR, but would you mind adding `ap-northeast-3` to the exclude list [here](https://github.com/hashicorp/terraform-aws-vault/blob/master/test/vault_main_test.go#L105)? I can then re-run the tests.

**Comment by queglay** [comment]
No worries, I've just added ap-northeast-3 to exclude list

**Comment by brikis98** [comment]
Thanks! I'll re-run the tests now.

**Comment by brikis98** [comment]
Tests passed! Merging now.

**Pull request merged**